### PR TITLE
Add Contributing doc

### LINKED ISSUE #81: Add Contributing.md documentation
It'll be nice, for new contributors to have a step they can follow to set up the repository locally
and how they can format their pull request to facilitate the triages process (like description, changes, fix) etc...

### PR DESCRIPTION
**Pull request by JFancis**
Hi, 
I just finished writing the contribution file Beta

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+### Attribution
```
Comment (JeanPhilippeKernel) [inline Contributing.md]: Can we remove this attribution part ??

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+
+Z-ENGINE - Contributing
```
Comment (JeanPhilippeKernel) [inline Contributing.md]: I don't think we need it so you can remove it

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+
+Z-ENGINE - Contributing 
+
+# Contributing
```
Comment (JeanPhilippeKernel) [inline Contributing.md]: nit : Contributing to ZEngine

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+1. Make sure to transmit that your modification
+
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## Code of Conduct
```
Comment (JeanPhilippeKernel) [inline Contributing.md]: We have to remove/move the Code of conduct part to another doc  and maybe reference it here as a bullet point  of the pull request section

**Review by JeanPhilippeKernel — CHANGES_REQUESTED** [review]

**Review by JFancis — COMMENTED** [review]
the contributing file has been updated,
and Code Conduct Created.

**Review by JeanPhilippeKernel — APPROVED** [review]

**Comment by JeanPhilippeKernel** [comment]
Thanks for your help  !! 😉🎉

**Pull request merged**